### PR TITLE
ELEMENTS-781: rethrow page provider fetch error

### PR DESCRIPTION
**Pull request by nmpcunha**
Pipeline: https://qa.nuxeo.org/jenkins/job/web-ui-pipeline-ncunha/51/

### CODE REVIEW

File: nuxeo-page-provider.html [hunk 1]
```diff
@@ -341,14 +341,15 @@
...
             })
-            .catch((err) => {
-              this.dispatchEvent(new CustomEvent('response', {
+            .catch((error) => {
+              this.dispatchEvent(new CustomEvent('error', {
                 bubbles: true,
                 composed: true,
                 detail: {
-                  error: err,
+                  error,
```
Comment (guirenard) [inline nuxeo-page-provider.html]: Is this syntax expected? how do you access error project property when handling event details?

Comment (nmpcunha) [reply nuxeo-page-provider.html]: I think it is. We will have something like`{ detail: { error: {...} }`

**Comment by guirenard** [comment]
:+1:

File: nuxeo-page-provider.html (hunk 1 above)
Comment (Gabez0r) [reply nuxeo-page-provider.html]: Yes it is: https://eslint.org/docs/rules/object-shorthand

File: test/.eslintrc.json [hunk 2]
```diff
@@ -9,5 +9,8 @@
     "expect": true,
     "waitChanged": true,
     "waitForEvent": true
+  },
+  "rules": {
+    "no-unused-expressions": "off"
```
Comment (Gabez0r) [inline test/.eslintrc.json]: I don't think we should disable this rule for the entire project. Please disable the rule for the particular line, or file if there are multiple lines at stake.

File: test/nuxeo-page-provider.html [hunk 3]
```diff
@@ -97,6 +103,56 @@
...
+                500,
+                responseHeaders.json,
+                '{"entity-type":"exception", "message":"Internal Server Error", "status": 500}',
+              ],
+            );
+            const pp = fixture('nx-pp');
+            return pp.fetch().then(() => {
+              throw new Error('Expected to have an error response!');
+            }).catch((error) => {
+              expect(error).to.exist;
```
Comment (Gabez0r) [inline test/nuxeo-page-provider.html]: If this is where the lint fails:
```suggestion
              expect(error).to.exist; // eslint-disable-line no-unused-expressions
```

File: test/.eslintrc.json (hunk 2 above)
Comment (nmpcunha) [reply test/.eslintrc.json]: I'll remove it from global tests scope and disable the rule only for that specific test.
Later on we can revisit this topic if other tests face the same need.

**Comment by Gabez0r** [comment]
👍

**Pull request merged**